src/ausearch-lol.c

```diff
@@ -35,6 +35,10 @@
 static int ready = 0;
 event very_first_event;
 
+// End of Event timeout value (in seconds). This is over-ridden by auditd-conf's end_of_event_timeout configuration item
+time_t eoe_timeout = EOE_TIMEOUT;
```

Comment:
Should this be a static variable and we have a set function that ausearch/report calls unconditionally to initialize this with either the config file value, the passed value, or the default value?

Reply:
If this is the philosophy wanted, then I can do it. Please advise.

Reply:
I think it would be better to have the source of truth in one place and it gets set from outside by the app which should consider which of the three the values it is and make it so.

Reply:
Done.